sequence_processing_pipeline/util.py

```diff
@@ -1,7 +1,18 @@
 import re
 
 
-PAIR_UNDERSCORE = (re.compile(r'_R1_'), '_R1_', '_R2_')
+# PAIR_UNDERSCORE = (re.compile(r'_R1_'), '_R1_', '_R2_')
+# The above will truncate on the first _R1_ found, which only works when _R1_
+# or _R2_ appears exactly once in a file path. When the wet-lab incorporates
+# these same strings in their sample-names as descriptive metadata, this
+# assumption is broken. For all raw fastq files being used as input into
+# NuQCJob, we can assume they end in the following convention. Per Illumina
+# spec, all fastq files end in _001 and we preserve this convention even at
+# the cost of renaming output files from TRIntegrateJob.
+# PAIR_DOT is kept as is, but may be removed later because for the purposes of
+# SPP, no input should ever be named with dots instead of underscores.
+PAIR_UNDERSCORE = (re.compile(r'_R1_001.fastq.gz'),
+                   '_R1_001.fastq.gz', '_R2_001.fastq.gz')
```

Comment:
I don't think this is safe for all data but it might be fine for TellSeq, no?

Reply:
Similar to the issue w/Commands. It should be fine for what we're currently using NuQCJob() for, but if we're going to create another pipeline then we have to move off of using glob to sort R1 and R2 files into adjacent elements in a list.